.snyk

```diff
@@ -0,0 +1,10 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.22.1
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-GOLANG-GITHUBCOMDGRIJALVAJWTGO-596515:
+    - '*':
+        reason: None Given
+        expires: 2021-12-22T16:25:33.572Z
+        created: 2021-11-22T16:25:33.580Z
```

Comment:
Can you give a bit of context about this ignore rule ?

Reply:
Yeah sure, let me update the `reason` field